shell/platform/darwin/macos/BUILD.gn

```diff
@@ -80,6 +80,7 @@ source_set("flutter_framework_source") {
   cflags_objcc = [ "-fobjc-arc" ]
 
   libs = [
+    "AudioToolbox.framework",
```

Comment:
NSBeep is AppKit, so won't require this.

Reply:
Yeah that would be better.

Reply:
`NSBeep` definitely seems better... why would anyone ever go with `AudioServicesPlayAlertSound*`?

Reply:
... it sounds like it may work on iOS?